src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/TokenManagement.java

```diff
@@ -0,0 +1,53 @@
+package com.nha.abdm.wrapper.hrp.mongo.tables;
+
+import org.apache.tomcat.util.json.Token;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.Date;
+
+@Document(collection = "tokenManagement")
+public class TokenManagement {
```

Comment:
class can be called `Token` and collection can be renamed as `tokens`

Reply:
Done re-naming.